regression/abstraction/Makefile

```diff
@@ -0,0 +1,38 @@
+default: tests.log
```

Comment:
Please can this be `regression/replication-reducing-abstraction/Makefile` or similar? Just `abstraction` is too ambiguous.

Reply:
Ok, I will change it to rr-abstraction.

Reply:
Changed to rr-abstraction.